Document need to install JDK 8 on Silicon Macs (#3059)
junit-team/junit5: README.md

You need [JDK 17] to build JUnit 5. [Gradle toolchains] are used to detect and
potentially download additional JDKs for compilation and test execution.

Note: If you're working on a machine with an Apple Silicon processor, you'll need to

> download and install a JDK 8 distribution manually (or via [SDKMAN!] or [asdf]),
> e.g. the one from [Zulu] or [Liberica].

Reviewer: ```suggestion
download and install a JDK 8 distribution manually (or via [SDKMAN!] or [asdf]) --
for example, one from [Zulu] or [Liberica].
```

[JaCoCo]: https://www.eclemma.org/jacoco/
[Javadoc]: https://junit.org/junit5/docs/current/api/
[JDK 17]: https://adoptium.net/archive.html?variant=openjdk17&jvmVariant=hotspot
[Liberica]: https://bell-sw.com/pages/downloads/
[Release Notes]: https://junit.org/junit5/docs/current/release-notes/
[Samples]: https://github.com/junit-team/junit5-samples
[SDKMAN!]: https://sdkman.io/jdks
[StackOverflow]: https://stackoverflow.com/questions/tagged/junit5
[User Guide]: https://junit.org/junit5/docs/current/user-guide/

> [Zulu]: https://www.azul.com/downloads/?version=java-8-lts&os=macos&architecture=arm-64-bit&package=jdk

Reviewer: ```suggestion
[Zulu]: https://www.azul.com/downloads/?version=java-8-lts&os=macos&architecture=arm-64-bit&package=jdk#download-openjdk
```
